config.go

```diff
@@ -73,11 +54,17 @@ func ValidateLogConfig(origin string, rootsPemFile string, rejectExpired bool, r
 		return nil, errors.New("empty origin")
 	}
 
+	// Load the trusted roots.
 	if rootsPemFile == "" {
 		return nil, errors.New("empty rootsPemFile")
 	}
+	roots := x509util.NewPEMCertPool()
+	if err := roots.AppendCertsFromPEMFile(rootsPemFile); err != nil {
+		return nil, fmt.Errorf("failed to read trusted roots: %v", err)
+	}
 
 	// Validate signer that only ECDSA is supported.
+	// TODO(phboneff): if this is a library this should also allow RSA as per RFC6962.
```

Comment:
Devil's avocado: if it supports ECDSA then that's enough to build RFC6962-compliant logs; there's no _requirement_ to support RSA (or, would we _recommend_ anyone to use RSA for a new log, these days?)